Framework/CMakeLists.txt

```diff
@@ -66,7 +66,7 @@ endif()
 message(STATUS "Deduced git SHA: ${GIT_SHA1}")
 
 # Include ldmx-sw version in the Run Header
-file(READ ${PROJECT_SOURCE_DIR}/../.github/actions/validate/gold_label LDMXSW_VERSION)
+file(READ ${PROJECT_SOURCE_DIR}/../VERSION LDMXSW_VERSION)
 string(STRIP "${LDMXSW_VERSION}" LDMXSW_VERSION)
```

Comment:
```suggestion
```
don't need this here anymore since it is defined in the central CMakeLists.txt and the value is inherited